django_testing_utils/mixins.py

```diff
@@ -186,12 +186,12 @@ def update_object(obj: models.Model, *args: Any, **kwargs: Any) -> None:
         """ Update django model object in database only."""
         args_iter = iter(args)
         kwargs.update(dict(zip(args_iter, args_iter)))
-        obj._meta.model.objects.filter(pk=obj.pk).update(**kwargs)
+        obj._meta.model.objects.filter(pk=obj.pk).update(**kwargs)  # type: ignore[attr-defined]
 
     @staticmethod
     def reload(obj: M) -> M:
         """ Fetch same object from database."""
-        return obj._meta.model.objects.get(pk=obj.pk)
+        return obj._meta.model.objects.get(pk=obj.pk)  # type: ignore[attr-defined]
```

Comment:
What is the error ignored here?

Reply:
Removed

Reply:
![image](https://github.com/just-work/django-testing-utils/assets/157370441/5f76b206-02dd-415b-b220-df42848e3380)
Return

Reply:
It's almost basic functionality of django-stubs :( 

Add a comment to source code and create an issue in GH. Then we a done here.

Reply:
Done